src/ServiceContainer/WebApiExtension.php

```diff
@@ -54,6 +54,9 @@ public function configure(ArrayNodeDefinition $builder)
                 ->scalarNode('base_url')
                     ->defaultValue('http://localhost')
                     ->end()
+                ->booleanNode('verify')
+                    ->defaultValue(true)
```

Comment:
Because we are supposed to test our own API, which is sometimes self-signed in dev, would it be a bad practice to set `->defaultValue(false)` ?